ext/openssl/openssl.stub.php

```diff
@@ -71,6 +75,10 @@ function openssl_csr_sign(OpenSSLCertificateSigningRequest|string $csr, OpenSSLC
 /** @param OpenSSLAsymmetricKey $private_key */
 function openssl_csr_new(array $distinguished_names, &$private_key, ?array $options = null, ?array $extra_attributes = null): OpenSSLCertificateSigningRequest|false {}
 
+/**
+ * @return array<int, string|array>|false
```

Comment:
This doesn't look right ... shouldn't the keys be `string`?

Reply:
You might want to build with `CFLAGS="-DZEND_VERIFY_FUNC_INFO=1"`.

Reply:
yes, you are right!